layers/meta-opentrons/recipes-images/images/opentrons-ot3-image.bb

```diff
@@ -107,6 +121,8 @@ do_make_rootfs_changes() {
 
     # add the VERSION.json file
     cat ${DEPLOY_DIR_IMAGE}/VERSION.json > ${IMAGE_ROOTFS}/etc/VERSION.json
+    # copy the release notes to the output dir
+    cat ${IMAGE_ROOTFS}/etc/release-notes.md > ${DEPLOY_DIR_IMAGE}/release-notes.md
```

Comment:
maybe `${IMAGE_ROOTFS}${sysconfdir}`? that's what we're using in the robot server recipe